src/inputs/checkbox.scss

```diff
@@ -86,7 +86,7 @@
 
   &:disabled {
     --_iui-checkbox-svg-color: var(--iui-icons-color-actionable-disabled);
-    --_iui-checkbox-border-color: var(--iui-color-background-disabled);
+    --_iui-checkbox-border-color: var(--iui-color-background-border);
```

Comment:
Do we want to exclude it from visibility checkbox? This looks a bit cluttered 
<img width="76" alt="image" src="https://user-images.githubusercontent.com/9084735/164029241-7bfd27ce-bb63-44e9-841a-6a4608192cbf.png">

Reply:
Oh, you could avoid the duplicate variable declarations by using `:enabled` selector inside `:where` (as it does not increase specificity)

<img width="1248" alt="image" src="https://user-images.githubusercontent.com/9084735/164042529-f4c3e830-5569-4165-a2ad-70adce1ebb63.png">

Reply:
To me it would be a bit more clearer what colours are used in visibility checkbox when variables are duplicated. But it's not good to duplicate code. 
Weren't we removing `:enabled` selector from buttons when refactoring them?

Reply:
We were removing `:enabled` from button for two main reasons:
- specificity (which is solved by `:where`)
- polymorphic buttons (e.g. rendered as `<a>` which does not support `:disabled` or `:enabled`)

I'm fine with duplicate code here, it's only two lines.